sdk/formrecognizer/Azure.AI.FormRecognizer/src/ContentType.cs

```diff
@@ -1,23 +1,30 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Azure.Core;
+
 namespace Azure.AI.FormRecognizer.Models
 {
     /// <summary>
     /// Form content type for local files.
     /// </summary>
-    public enum FormContentType
+    [CodeGenSchema("ContentType")]
```

Comment:
is application/json the default?

Reply:
This is used in the stream overloads to specify the type of the input form, so there's no application/json.  It's generated from the enum in the Swagger ... does Python have something different?